resources/public/api/conf/1.0/application.yaml

```diff
@@ -1,47 +1,536 @@
-openapi: "3.0.3"
-
+openapi: 3.0.3
 info:
-  version: 1.0.0
-  title: Hello World
-
+  title: Excise Movement Control System API (Discovery/Proposal)
+  description: |-
+    This is a very rough draft of an idea for a potential replacement for the current [EMCS SOAP API](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/xml/Excise%20Movement%20Control%20System)
+  contact: {}
+  version: "0.0.0"
 servers:
-  - url: https://test-api.service.hmrc.gov.uk/hello-world
+  - url: https://test-api.service.hmrc.gov.uk
     description: Sandbox
-  - url: https://api.service.hmrc.gov.uk/hello-world
+  - url: https://api.service.hmrc.gov.uk
     description: Production
-
-components:
-  parameters:
-    acceptHeader:
-      name: Accept
-      in: header
-      schema:
-        type: string
-        enum: [
-          "application/vnd.hmrc.1.0+json",
-          "application/vnd.hmrc.1.0+xml"
-        ]
-      required: true
-
+tags:
+  - name: Movements
+    description: Create, amend or cancel an Excise Movement
+  - name: Traders
+    description: Information relating to Excise Traders
 paths:
-  /hello-world:
+  /customs/excise/movements:
+    post:
+      tags:
+        - Movements
+      summary: Submit a new Draft Excise Movement
+      description: Submit a new Draft Excise Movement. </br></br>**Existing SOAP API equivalent:** SubmitDraftMovement </br></br>**Note:** You can only submit a Draft Excise Movement if the Consignor Id matches an Excise Number in the bearer token that you use for API authentication. Otherwise a 403 Forbidden error is thrown.
```

Comment:
More a query to confirm my knowledge: Excise Number is the same thing as Excise Reference Number right?

Reply:
I have made them all ERNs and described the acronym